Source/Module/Login/LoginView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LoginView.swift
+//  GithubNotifications
+//
+//  Created by Chananchida F. on 9/8/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+
+    private let loginDescription: [TextContent] = [
+        .text("To generate a token, go to Github, "),
+        .link("personal access tokens", link: "https://github.com/settings/tokens"),
```

Comment:
<img width="778" alt="Screen Shot 2021-09-09 at 18 34 01" src="https://user-images.githubusercontent.com/19943832/132678551-dae91019-217b-4e13-933d-fbf5e9f59d4c.png">

We have markdown support, but still beta 😭

Reply:
https://developer.apple.com/documentation/foundation/attributedstring